modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/upgrade/v4_1_0/util/JournalArticleTable.java

```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package null;
+package com.liferay.journal.internal.upgrade.v4_1_0.util;
```

Comment:
strange, do you had to modified it manually?

Reply:
Yes... It was weird because everything else was generated right